api/src/Entity/Parchment.php

```diff
@@ -17,7 +17,8 @@ class Parchment
      *
      * @ORM\Column(type="integer")
```

Comment:
`type="uuid"` here and in other places

Reply:
As the UUID generation is done by the API and not by the database, shouldn't it be `string`?

Reply:
I'm almost sure but if you're right, you don't even need to declare the extra type like done in `api/config/packages/ramsey_uuid_doctrine.yaml` as you're not using it... The generator will give you an Uuid object, and you need to tell doctrine how to map it properly I think